x-pack/test/functional_gen_ai/inference/tests/chat_complete.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lastValueFrom, toArray } from 'rxjs';
+import expect from '@kbn/expect';
+import { supertestToObservable } from '@kbn/sse-utils-server';
+import type { AvailableConnectorWithId } from '@kbn/gen-ai-functional-testing';
+import type { FtrProviderContext } from '../ftr_provider_context';
+
+export const chatCompleteSuite = (
+  { id: connectorId, actionTypeId: connectorType }: AvailableConnectorWithId,
+  { getService }: FtrProviderContext
+) => {
+  const supertest = getService('supertest');
+
+  describe('chatComplete API', () => {
+    describe('streaming disabled', () => {
+      it('returns a chat completion message for a simple prompt', async () => {
+        const response = await supertest
```

Comment:
Let's remember to update this suite to account for permissions once we're aligned on our authorization model.